puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/deployPuppet.groovy

```diff
@@ -1,14 +1,17 @@
 pipeline {
     agent { label 'admin && sshkey' }
-
     options {
         timestamps()
         disableConcurrentBuilds()
         ansiColor('xterm')
     }
-
```

Comment:
Oh sorry, do you mind if we keep these blank lines? I think at least myself and @evgeni find them to be useful for visually separating out major code sections